code/modules/client/preference/loadout/loadout_gloves.dm

```diff
@@ -5,4 +5,12 @@
 
 /datum/gear/gloves/fingerless
 	display_name = "Fingerless Gloves"
-	path = /obj/item/clothing/gloves/fingerless
\ No newline at end of file
+	path = /obj/item/clothing/gloves/fingerless
+
+/datum/gear/gloves/silverring
+	display_name = "Silver ring"
+	path = /obj/item/clothing/gloves/ring/silver
+
+/datum/gear/gloves/goldring
+	display_name = "Gold ring"
+	path = /obj/item/clothing/gloves/ring/gold
```

Comment:
abuseable

Reply:
Probably fine--I forget what I even opposed this for.

Reply:
OK